07-asymmauth.tex

```diff
@@ -191,8 +191,8 @@ \subsection{Schlüssel- und Signaturerzeugung}
 Zahl $e \in \{1, \dots, p - 1\}$ gewählt,
 wobei $p=|\G|$. Damit berechnet man 
 \begin{align*}
-a &:= g^e \in \mathbb{G}\\
-b &:= (M - a \cdot x) \cdot e^{-1} \mod p
+a &:= g^e \mod p \in \mathbb{G}\\
```

Comment:
Klar, dann ist das im Commit aber nicht richtig, oder? Denn hier rechnen wir ja a aus was in G liegen soll. Also mod N (N prim)
und b &:= (M - a \cdot x) \cdot e^{-1} \mod (p-1) müsste b &:= (M - a \cdot x) \cdot e^{-1} \mod p
sein, da wir ja hier modulo der Gruppenordnung rechnen (also p)

Das kann sein, dass das an anderer stelle war. Denn irgendwo im Skript steht etwas von Z_p (also die Gruppe modulo einer Primzahl p) und danach wird p als die Gruppenordnung definiert.